### DISCUSSION TITLE
Interest in creating a *.deb package

### DISCUSSION THREAD

**Discussion by allnamesaretakeningithub** [pystardust/ytfzf#286] [Ideas]
I can assist in creating *.deb package file for `ytfzf`. that could maybe be shared on git. I am not a debian maintainer, so I cannot upload it there.

This is a POC of a deb package (github does not allow to upload files with the *.deb extension so I zipped it  [ytfzf.zip](https://github.com/pystardust/ytfzf/files/6651790/ytfzf.zip)

This package has no `ueberzug` because since it is not in the debian repos, needs install via pip, that can be done in a *deb package as post install, but it would not be compliant.

Also there is the case of what contact info to put in the deb package.

Anyhow if there is interest, let me know

**Comment by pystardust** [comment]
It would be great if you could package it on debian. You could leave a link to the issue page. Shouldn't it be the contact of  the maintainer of the debian package?

**Reply by allnamesaretakeningithub** [reply]
> It would be great if you could package it on debian. 

Making a deb package and being able to upload it to the debian repos are/can be two different tasks.

I can make a debian package that installs. 

To upload it to the debian repos I would need to find a debian developer that would sponsor that package.

> You could leave a link to the issue page. 

I can open an ITP (intend to package) in the debian bug tracker and look if there is an interested sponsor.

> Shouldn't it be the contact of the maintainer of the debian package?

Currently in the package I made I am the maintainer there. But that could change.

ok,  my take from your reply is that you would be more interested in being able to upload the package to the debian repos?
